### PR TITLE
✨ harmonizing controller-manager name

### LINKED ISSUE #1727: feature: Finish harmonizing on "kubestellar controller-manager"
### Feature Description

We still have references to "kubestellar-operator".  For example, in the container image and Helm chart names.

### Proposed Solution

Consistently use "kubestellar operator" or consistently use "kubestellar controller-manager". Do not use a mixture of the two.

### Want to contribute?

- [ ] I would like to work on this issue.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by pdettori**
## Summary

- harmonizing controller-manager name
- rename chart directory 

Fixes #1727

### CODE REVIEW

File: docs/content/Coding Milestones/PoC2023q1/commands.md [hunk 1]
```diff
@@ -382,7 +382,7 @@ start](#kubestellar-start) command above.
 ### Fetch a log from a KubeStellar runtime container
 
 {%
-   include-markdown "../../../../core-helm-chart/README.md"
+   include-markdown "../../../../chart/README.md"
```
Comment (MikeSpreitzer) [inline docs/content/Coding Milestones/PoC2023q1/commands.md:385]: This document is obsolete, updating it for this one change would be confusing.

File: docs/content/Contribution guidelines/operations/packaging-and-versioning.md [hunk 2]
```diff
@@ -77,22 +77,22 @@ This Helm chart describes a deployment of the KubeStellar core
 components (a space provider, KubeStellar controllers, initialization) as
 workload for a Kubernetes cluster.
 
-The source of this Helm chart is in the `core-helm-chart` subdirectory
```
Comment (MikeSpreitzer) [inline docs/content/Contribution guidelines/operations/packaging-and-versioning.md:80]: This document is obsolete. Updating it for this one change would be confusing. See docs/content/v02.0/packaging.md.

File: docs/content/Contribution guidelines/operations/release-management.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ change the version in the following line:
 syncer_image="quay.io/kubestellar/syncer:{{ config.ks_next_tag }}"
 ```
 
-### Update the core-helm-chart 'Chart.yaml' and 'values.yaml' files with a reference to the new version of the KubeStellar Helm chart version
+### Update the chart 'Chart.yaml' and 'values.yaml' files with a reference to the new version of the KubeStellar Helm chart version
```
Comment (MikeSpreitzer) [inline docs/content/Contribution guidelines/operations/release-management.md:33]: This document is obsolete. Please do not muddy the waters by trying to change it partially. The new release procedure is being developed in Issue #1732 and PR #1758 .

File: hack/make-release-platform-archive.sh [hunk 4]
```diff
@@ -55,7 +55,7 @@ make ${scope_part}build OS="$target_os" ARCH="$target_arch"
 echo $'#!/usr/bin/env bash\necho' ${kcpe_version} > bin/kubestellar-release
 chmod a+x bin/kubestellar-release
 mkdir -p build/release
-tar czf "build/release/$archname" --exclude bin/.gitignore bin config core-helm-chart README.md LICENSE
+tar czf "build/release/$archname" --exclude bin/.gitignore bin config chart README.md LICENSE
```
Comment (MikeSpreitzer) [inline hack/make-release-platform-archive.sh:58]: This script is obsolete, modifying it now would just be confusing.

File: docs/content/Coding Milestones/PoC2023q1/commands.md (hunk 1 above)
Comment (pdettori) [reply docs/content/Coding Milestones/PoC2023q1/commands.md]: ok, change undone

File: docs/content/Contribution guidelines/operations/packaging-and-versioning.md (hunk 2 above)
Comment (pdettori) [reply docs/content/Contribution guidelines/operations/packaging-and-versioning.md]: change undone

File: docs/content/Contribution guidelines/operations/release-management.md (hunk 3 above)
Comment (pdettori) [reply docs/content/Contribution guidelines/operations/release-management.md]: change undone

File: hack/make-release-platform-archive.sh (hunk 4 above)
Comment (pdettori) [reply hack/make-release-platform-archive.sh]: undone

**Comment by MikeSpreitzer** [comment]
I would like to make this change as part of working out the new release process, so that we do not get ourselves stuck again with problematic GitHub package ownership in ghcr.io.

File: chart/templates/controller-manager.yaml [hunk 5]
```diff
@@ -202,8 +202,7 @@ spec:
         - --leader-elect
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
-        - -v=2
```
Comment (MikeSpreitzer) [inline chart/templates/controller-manager.yaml:205]: Oh, I did not notice this until reviewing #1779.

**Pull request merged**